docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -53,7 +55,36 @@ You can use YBA Installer only if you are about to perform a new install. Curren
 ## Prerequisites
 
 - Ensure your machine satisfies the [minimum requirements](../../prerequisites/installer/).
-- For production deployments, root privileges (or equivalent sudo access) are required for some YBA Installer commands. (You can use YBA Installer without root access, but this is not recommended for production; refer to [Non-root installation](#non-root-installation).)
+- For production deployments, sudo permissions are required for some YBA Installer commands. (You can use YBA Installer without sudo access, but this is not recommended for production; refer to [Non-sudo installation](#non-sudo-installation).)
+
+## Quick start
+
+To install YugabyteDB Anywhere using YBA Installer, do the following:
+
+1. Obtain your license from Yugabyte Support.
+1. Download and extract the YBA Installer by entering the following commands:
+
+    ```sh
+    $ wget https://downloads.yugabyte.com/releases/{{<yb-version version="stable" format="long">}}/yba_installer_full-{{<yb-version version="stable" format="build">}}-linux-x86_64.tar.gz
+    $ tar -xf yba_installer_full-{{<yb-version version="stable" format="build">}}-linux-x86_64.tar.gz
+    $ cd yba_installer_full-{{<yb-version version="stable" format="build">}}/
+    ```
+
+1. Run a preflight check to ensure your environment satisfies the requirements. Respond with `y` when prompted to create a default configuration.
+
+    ```sh
+    $ sudo ./yba-ctl preflight
+    ```
+
+1. If there are no issues, install the software, providing your license.
+
+    ```sh
+    $ sudo ./yba-ctl install -l /path/to/license
+    ```
+
+After the install succeeds, you can immediately start using YBA.
+
```

Comment:
If the install fails  due to permissions or lack of "sudo" priviledges, it can be re-tried after doin a 
"yba-ctl clean all" to remove all traces of the previous install attempt.